docs/Reference/Decorators.md

```diff
@@ -155,40 +148,39 @@ fastify.listen({ port: 3000 }, (err, address) => {
 })
 ```
 
-Note: using an arrow function will break the binding of `this` to the
-`FastifyInstance`.
+Using an arrow function breaks the binding of `this` to
+the `FastifyInstance`.
```

Comment:
I understand why it's there, but I don't like that this is repeated three times across this doc.
Can't think of a nicer way to express it though.

Reply:
Frankly, so many people writing JavaScript don't understand this that it's best to call it out as many times as possible. If we manage to get these people to even read the docs, they are only going to read the specific section they need to solve their problem, and are mostly just going to copy and paste things.

Reply:
Good point. p.s. thanks again for reviewing these!